templates/taxi/car_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Create Car</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+{#    {{ form.as_p }}#}
+    <input type="submit" value="Submit">

```

Comment:
Consider adding Bootstrap classes to the submit button for styling consistency, such as `class="btn btn-primary"`.